sunpy/instr/tests/test_lyra.py

```diff
@@ -83,31 +82,21 @@ def test_split_series_using_lytaf():
 
 
 @pytest.fixture
-def get_lyradata(dtype):
-    if dtype == 'lc':
-        # Create sample LYRALightCurve
-        lyrats = lightcurve.LYRALightCurve.create("2014-01-01")
-        lyrats.data = pandas.DataFrame(index=TIME,
-                                       data={"CHANNEL1": CHANNELS[0],
-                                             "CHANNEL2": CHANNELS[1],
-                                             "CHANNEL3": CHANNELS[0],
-                                             "CHANNEL4": CHANNELS[1]})
-    else:
-        # Create sample TimeSeries
-        lyrats = timeseries.TimeSeries(
-            os.path.join(rootdir, 'lyra_20150101-000000_lev3_std_truncated.fits.gz'),
-            source='LYRA')
-        lyrats.data = pandas.DataFrame(index=TIME,
-                                       data={"CHANNEL1": CHANNELS[0],
-                                             "CHANNEL2": CHANNELS[1],
-                                             "CHANNEL3": CHANNELS[0],
-                                             "CHANNEL4": CHANNELS[1]})
+def ts():
+    # Create sample TimeSeries
+    lyrats = timeseries.TimeSeries(
+        os.path.join(rootdir, 'lyra_20150101-000000_lev3_std_truncated.fits.gz'),
+        source='LYRA')
+    lyrats.data = pandas.DataFrame(index=TIME,
+                                   data={"CHANNEL1": CHANNELS[0],
+                                         "CHANNEL2": CHANNELS[1],
+                                         "CHANNEL3": CHANNELS[0],
+                                         "CHANNEL4": CHANNELS[1]})
     return lyrats
 
 
 @pytest.mark.remote_data
-@pytest.mark.parametrize('dtype', ['ts', 'lc'])
-def test_remove_lytaf_events_from_timeseries(dtype):
+def test_remove_lytaf_events_from_timeseries(ts):
     """Test if artefacts are correctly removed from a TimeSeries.
```

Comment:
This docstring probably needs updating so it's clear what the input is.  The mention of the ```LYRALightCurve``` can probably go too.